src/server/api/schema.js

```diff
@@ -3408,6 +3408,34 @@ const rootMutations = {
         .where({ email: userEmail })
         .update({ is_superadmin: superAdminStatus });
 
+      return true;
+    },
+    editOrganizationActive: async (
+      _root,
+      { organizationId, active },
+      { user }
+    ) => {
+      await superAdminRequired(user);
+
+      if (active) {
+        await r
+          .knex("organization")
+          .where({ id: organizationId })
+          .update({ deleted_at: null, deleted_by: null });
+      } else {
+        await r
+          .knex("organization")
+          .where({ id: organizationId })
+          .update({ deleted_at: r.knex.fn.now(), deleted_by: user.id });
+
+        // Suspend all users except owners (so superadmins can access these orgs and their settings)
```

Comment:
Question: One of the common reasons we get requests to deactivate orgs is because owners who are in many orgs (but are not superadmins) want to declutter the big list of orgs that will pop up in a few places for them in Spoke. Is there a different way of handling this so that we can still achieve what they're looking for as well?

Reply:
This still achieves that just fine! The list isn't displayed in many places if the organization has a `deleted_at` set, https://github.com/politics-rewired/Spoke/pull/1348/files#diff-6167f7d4b5d9664e676a5d148c2a2c316b56085d6ac43620eaab1aeb07e02e96R3522 this change when doing a query of organizations with `organizations(active: true)` will not list any organizations that aren't active, but if the owner knew the organization id, they could still go back using the URL and access all the campaign data or any other data if they still wanted exports, or similar.

Reply:
Incoming!

**Do not suspend or delete users**
helper text on hover:
_Unsuspended users may be able to navigate to the organization directly. The organization will not appear on users' organization list._
**Suspend all users**
helper text on hover:
_Users who were suspended as part of shutdown can optionally be unsuspended upon reactivation of the organization._
**Delete all users and shut down organization**
helper text on hover: (let us know if this needs to be shortened 
_User records will be deleted. Deleted users cannot be reinstated upon reactivation of the organization. Users will need to be sent a join link  to rejoin the organization._

_This option should be used to fully shut down the organization and trigger the release of phone numbers and finalized billing for the organization. Owners who want to select the first two options but also not be billed for inbounds can send an email requesting that phone numbers be sold._


**Notes for implementing**
Option 3 automatically notifies us (by email?) that they shut down the org so that we can complete the process of selling off phone numbers/completing full shutdown.

Reactivation of an organization where users were suspended should have a toggle to optionally unsuspend users who were suspended as part of the deactivation process (so users who are manually suspended by admins etc. are still suspended)

Also @bchrobot what were you thinking the use case is for removing all `user_organization` records? For us it seems like it would be that you're removing all texters from the org, and if you reactivate the org for whatever reason it's like you're starting over.

Reply:
<img width="640" alt="Screenshot 2022-08-18 at 3 53 13 PM" src="https://user-images.githubusercontent.com/367605/185373096-1d4c43a8-e7c1-4839-9d28-f80061735eba.png">
<img width="692" alt="Screenshot 2022-08-18 at 3 53 17 PM" src="https://user-images.githubusercontent.com/367605/185373116-28946a81-2ddb-4e07-92df-3055fa651d38.png">
<img width="676" alt="Screenshot 2022-08-18 at 3 53 20 PM" src="https://user-images.githubusercontent.com/367605/185373124-fa3e1685-eba3-4065-8048-d54408d20d44.png">


@politics-rewired/organizing feedback please!

Reply:
@hiemanshu Can we change the last option to read "Delete all users (including owners) and shut down organization?

Reply:
Sounds good, last question @politics-rewired/organizing, what email ID should the email come to, asking for the org shutdown? support email?

Reply:
@hiemanshu yes, send to support!